hi-ml/src/health_ml/run_ml.py

```diff
@@ -225,11 +226,9 @@ def after_ddp_cleanup(self, old_environ: Dict) -> None:
         # We can now use the global_rank of the Lightning model, rather than environment variables, because DDP has set
         # all necessary properties.
         if self.container.model.global_rank != 0:
-            logging.info(f"Terminating training thread with rank {self.container.model.global_rank}.")
+            print_message_with_rank_pid(f"Terminating training thread with rank {self.container.model.global_rank}.")
```

Comment:
this wasn't logged at all because PL restricts logs from rank 0 only

Reply:
At present, print_message... also prints out a big DEBUG sign, which I feel will look a bit out of place here. If we think the information about killing process is important, then I'd remove that DEBUG sign

Reply:
good point, will add a level argument so that we can specify if it's a debug or info or warning